app/view_utils/marketplace_helper.rb

```diff
@@ -6,4 +6,22 @@ def google_maps_key(cid)
     community_key ? community_key : APP_CONFIG.google_maps_key
   end
 
+  def style_customizations_map(community)
+    color1 = community.custom_color1&.downcase || "a64c5d"
+    color2 = community.custom_color2&.downcase || community.custom_color1&.downcase || "00a26c"
```

Comment:
Good point, they are currently in multiple places (I think). Have to check if it's easy to put them all in one place.